integration-tests/scaffolding/helpers.ts

```diff
@@ -114,11 +115,17 @@ export async function fundKeypair(source: KeyringPair, dest: KeyringPair, amount
   await ExtrinsicHelper.transferFunds(source, dest, amount).signAndSend(nonce);
 }
 
-export async function createAndFundKeypair(amount = EXISTENTIAL_DEPOSIT, keyName?: string, devAccount?: KeyringPair, nonce?: number): Promise<KeyringPair> {
+export async function createAndFundKeypair(amount = EXISTENTIAL_DEPOSIT, keyName?: string, source?: KeyringPair, nonce?: number): Promise<KeyringPair> {
+  let default_source;
+  if (process.env.CHAIN_ENVIRONMENT == "rococo") {
+    default_source = rococoAccounts[0];
+  } else {
+    default_source = devAccounts[0];
+  }
```

Comment:
Might be worth hiding this if statement inside a function. Feels like it shouldn't be in more than one place.

Reply:
Agreed. We should probably retrieve the appropriate funding source from a function.